arches_rdm/etl_modules/migrate_rdm.py

```diff
@@ -0,0 +1,285 @@
+                uuid_generate_v4() as tileid,
+                true as passes_validation,
+                0 as nodegroup_depth,
+                'Scheme: top_concept_of' as source_description,
+                %s::uuid as loadid,
+                %s::uuid as nodegroupid,
+                'insert' as operation
+            from relations
+            where relationtype = 'hasTopConcept';
+        """, (CONCEPTS_TOP_CONCEPT_OF_NODEGROUP_ID, loadid, CONCEPTS_TOP_CONCEPT_OF_NODEGROUP_ID))
+
+        # Create broader relationships (derived from relations with 'narrower' relationtype)
+        cursor.execute("""
+           insert into load_staging(
+                value,
+                resourceid,
+                tileid,
+                passes_validation,
+                nodegroup_depth,
+                source_description,
+                loadid,
+                nodegroupid,
+                operation
+            )
+            select 
+                json_build_object(%s::uuid,
+                    json_build_object(
+                        'notes', '',
+                        'valid', true,
+                        'value', json_build_array(json_build_object('resourceId', conceptidfrom, 'ontologyProperty', '', 'resourceXresourceId', '', 'inverseOntologyProperty', '')),
+                        'source', conceptidfrom,
+                        'datatype', 'resource-instance-list'
+                    )
+                ) as value,
+                conceptidto as resourceinstanceid, -- map target concept's new resourceinstanceid to its existing conceptid
+                uuid_generate_v4() as tileid,
+                true as passes_validation,
+                0 as nodegroup_depth,
+                'Scheme: top_concept_of' as source_description,
+                %s::uuid as loadid,
+                %s::uuid as nodegroupid,
+                'insert' as operation
+            from relations
+            where relationtype = 'narrower';
+        """, (CONCEPTS_BROADER_NODEGROUP_ID, loadid, CONCEPTS_BROADER_NODEGROUP_ID))
+
+    def start(self, request):
+        load_details = {"operation": "RDM Migration"}
+        cursor = connection.cursor()
+        cursor.execute(
+            """INSERT INTO load_event (loadid, complete, status, etl_module_id, load_details, load_start_time, user_id) VALUES (%s, %s, %s, %s, %s, %s, %s)""",
+            (self.loadid, False, "running", self.moduleid, json.dumps(load_details), datetime.now(), self.userid),
+        )
+        message = "load event created"
+        return {"success": True, "data": message}
+
+    def write(self, request):
+        self.loadid = request.POST.get("loadid")
+        if models.Concept.objects.count() > 500:
+            response = self.run_load_task(self.userid, self.loadid)
+        else: 
+            response = self.run_load_task_async(request, self.loadid)
```

Comment:
If and else seem like they're swapped -- would expect async for higher volume of concepts